Makefile

```diff
@@ -29,29 +29,54 @@ show_bench:
 	@ cat $(BENCH)
 
 .PHONY: dev-clean
+dev-clean: CONTAINER_NAME=apisonator-dev
 dev-clean:
-	-docker kill apisonator-dev
-	-docker rm apisonator-dev
+	-docker kill $(CONTAINER_NAME)
+	-docker rm $(CONTAINER_NAME)
+
+.PHONY: dev-service-clean
+dev-service-clean: CONTAINER_NAME=apisonator-dev-service
+dev-service-clean:
+	-docker kill $(CONTAINER_NAME)
+	-docker rm $(CONTAINER_NAME)
 
 .PHONY: dev-clean-image
-dev-clean-image: IMAGE_NAME?=apisonator-dev
-dev-clean-image: dev-clean
+dev-clean-image: IMAGE_NAME=apisonator-dev
```

Comment:
are you sure you do want to use `?=` ?

Reply:
Hi,

You are right, there is no need to remove the ? . I did that because in the case of the variable CONTAINER_NAME i wanted it to be reevaluated always due to different rules (dev-clean, dev-service-clean) use that variable with different values.

I added a new commit to update this.

Thank you.